### PR TITLE
feat(pci.ai): make ai jobs deletion possible

### PR DESCRIPTION
**Pull request by abullet33**
| Question         | Answer
| ---------------- | ---
| Branch?          | feat/datatr-296-ai-dashboard-1.1
| Bug fix?         | yes
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #datatr-330 
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] Breaking change is mentioned in relevant commits

## Description
Allow customer to delete their ai jobs and uniformize ai tools.

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.routing.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    resolve: {
+      breadcrumb: () => null,
+      previousState: /* @ngInject */ ($transition$) => {
+        return $transition$.params().previousState;
+      },
+      jobId: /* @ngInject */ ($transition$) => $transition$.params().jobId,
+      job: /* @ngInject */ (PciProjectTrainingJobService, projectId, jobId) => {
+        return PciProjectTrainingJobService.get(projectId, jobId);
+      },
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.routing.js:21]: **niptick** : 
```suggestion
      job: /* @ngInject */ (PciProjectTrainingJobService, projectId, jobId) =>
        PciProjectTrainingJobService.get(projectId, jobId),
```

File: packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.routing.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    },
+    layout: 'modal',
+    params: {
+      previousState: null,
+    },
+    resolve: {
+      breadcrumb: () => null,
+      previousState: /* @ngInject */ ($transition$) => {
+        return $transition$.params().previousState;
+      },
```
Comment (darsene) [inline packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.routing.js:17]: **niptick :**
```suggestion
      previousState: /* @ngInject */ ($transition$) => $transition$.params().previousState,
```

File: packages/manager/modules/pci/src/projects/project/ai/apps/components/delete/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "pci_ai_apps_delete_success": "Votre app a été supprimée avec succès",
   "pci_ai_apps_delete_error": "Une erreur est survenue durant la suppression de l'app : {{ message }}",
   "pci_ai_apps_delete": "Suppression de l'app",
-  "pci_ai_apps_delete_common_delete": "Supprimer",
+  "pci_ai_apps_delete_common_delete": "Confirmer",
   "pci_ai_apps_delete_common_cancel": "Annuler",
-  "pci_ai_apps_delete_warning": "Êtes-vous sûr de vouloir supprimer l'app {{ appId }} ?"
+  "pci_ai_apps_delete_warning": "Êtes-vous sûr de vouloir supprimer l'app <strong>{{ appId }}</strong> ?"
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/ai/apps/components/delete/translations/Messages_fr_FR.json:7]: We must avoid html in translation files. Instead you can have the `strong` tag in the template like this:

`data-translate-values="{ appId: '<strong>' + $ctrl.app.name + '</strong>' }"`

File: packages/manager/modules/pci/src/projects/project/notebooks/components/delete-notebook/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -2,7 +2,7 @@
   "pci_notebook_delete_notebook_title": "Suppression du notebook",
   "pci_notebook_delete_notebook_submit_label": "Confirmer",
   "pci_notebook_delete_notebook_cancel_label": "Annuler",
-  "pci_notebook_delete_notebook_content": "Êtes-vous sûr·e de vouloir supprimer le notebook {{ notebookName }} ?",
+  "pci_notebook_delete_notebook_content": "Êtes-vous sûr de vouloir supprimer le notebook <strong>{{ notebookName }}</strong> ?",
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/notebooks/components/delete-notebook/translations/Messages_fr_FR.json:5]: Same here

File: packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/delete-notebook/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -1,6 +1,6 @@
 {
   "pci_notebooks_general_information_info_delete_notebook_title": "Suppression du notebook",
-  "pci_notebooks_general_information_info_delete_notebook_description": "Etes-vous sur de vouloir supprimer le notebook {{ name }} ?",
+  "pci_notebooks_general_information_info_delete_notebook_description": "Êtes-vous sûr de vouloir supprimer le notebook <strong>{{ name }}</strong> ?",
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/notebooks/dashboard/general-information/delete-notebook/translations/Messages_fr_FR.json:3]: Same here

File: packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.component.js [hunk 6]
```diff
@@ -0,0 +1,15 @@
+import controller from './delete-job.controller';
+import template from './delete-job.html';
+
+export default {
+  controller,
+  template,
+  bindings: {
+    goToJobs: '<',
+    goToJobInfo: '<',
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.component.js:9]: Do you need both functions ? I think you could bind a single `goBack` function in the `.component` file, with its definition in the parent module.
For exemple, this is done in the confirm-delete modale in databases:
- [modale component file](https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/confirm-delete/confirm-delete.component.js#L7): here we add a `goBack` function in the bindings
- [parent 1 routing file](https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/databases.routing.js#L191): this is the first place where the modale can be displayed, we define a `goBack` function in the parent that will be used in the modale component
- [parent 2 routing file](https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/confirm-delete-database/confirm-delete-database.routing.js#L22): second place where the modale can be displayed

That way, your modale component can be used in any place, and should correctly redirect you to the parent location without a query param

File: packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.controller.js [hunk 7]
```diff
@@ -0,0 +1,49 @@
...
+    this.isLoading = false;
+  }
+
+  goBack() {
+    if (this.previousState === 'info') {
+      this.goToJobInfo();
+    } else {
+      this.goToJobs();
+    }
+  }
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/delete-job.controller.js:18]: See comment on `delete-job.component.js`

File: packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/translations/Messages_fr_FR.json [hunk 8]
```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_training_jobs_delete_job_title": "Suppression du job",
+  "pci_projects_project_training_jobs_delete_job_description": "Etes-vous sûr de vouloir supprimer le job <strong>{{ name }}</strong> ?",
```
Comment (jperchoc) [inline packages/manager/modules/pci/src/projects/project/training/jobs/delete-job/translations/Messages_fr_FR.json:3]: Same as above, avoid html il translation files

**Review by jperchoc — APPROVED** [review]

**Review by qpavy — APPROVED** [review]

**Pull request merged**